openmc/model/model.py

```diff
@@ -2112,8 +2112,12 @@ def convert_to_multigroup(
         groups : openmc.mgxs.EnergyGroups or str, optional
             Energy group structure for the MGXS or the name of the group
             structure (based on keys from openmc.mgxs.GROUP_STRUCTURES).
+        nparticles : int, optional
```

Comment:
`n_particles` is more in line with our naming conventions but we can leave that for the next version. Just noting it here.